apps/etterlatte-saksbehandling-ui/client/src/store/reducers/PersonopplysningerReducer.ts

```diff
@@ -1,18 +1,28 @@
 import { createAction, createReducer } from '@reduxjs/toolkit'
-import { Personopplysninger } from '~shared/types/grunnlag'
+import { Personopplysning, Personopplysninger } from '~shared/types/grunnlag'
 
 export const setPersonopplysninger = createAction<Personopplysninger>('behandlingPersoner/set')
 export const resetPersonopplysninger = createAction('behandlingPersoner/reset')
 
-const initialState: { personopplysninger: Personopplysninger | null } = {
+const initialState: {
+  personopplysninger: Personopplysninger | null
+  avdoede: Personopplysning | null
+  gjenlevende: Personopplysning | null
+} = {
   personopplysninger: null,
+  avdoede: null,
+  gjenlevende: null,
 }
 
 export const personopplysningerReducer = createReducer(initialState, (builder) => {
   builder.addCase(setPersonopplysninger, (state, action) => {
     state.personopplysninger = action.payload
+    state.avdoede = action.payload.avdoede.find((po) => po) ?? null
```

Comment:
Tror denne er uheldig å ha med tanke på 2 avdøde i barnepensjon -- da ignorerer vi den andre avdøde. Alle steder der denne er brukt i barnepensjon blir dermed suspekt, med mindre vi vet at vi ikke trenger å forholde oss til 2 avdøde. For OMS er det greit nok tenker jeg, men vil kanskje gitt et mer spesifikt navn til egenskapen slik at man ikke bruker den uten videre.

Reply:
Har gjort om her nå, så kun Aktivitetsplikt benytter en reducer som er navngitt med OMS. Resten av komponentene er enten BP eller ikke-spesifikke hva jeg kan se. Det er ikke _så_ mange steder heller, så er vel ikke store problemet med litt ekstra "finne første element i lista på en trygg måte"-logikk i dem.